modules/core/src/main/java/org/apache/ignite/internal/processors/metastorage/persistence/DmsDataWriterWorker.java

```diff
@@ -69,7 +69,16 @@ public DmsWorkerStatus status() {
     private volatile ReadWriteMetastorage metastorage;
 
     /** */
-    private volatile boolean firstStart = true;
+    private volatile RunnableFuture<?> curTask;
```

Comment:
I see that curTask is used only locally in method "body"
I think it's better to move it into the method (don't see any reason to add volatile reads).

Reply:
Fixed.